.github/workflows/linux_cuda.yml

```diff
@@ -0,0 +1,162 @@
+name: Linux CUDA Plugin (Ubuntu 20.04, Python 3.11)
+on:
+  workflow_dispatch:
+  schedule:
+    # run daily at 00:00
+    - cron: '0 0 * * *'
+#  pull_request:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#  push:
+#    paths-ignore:
+#      - '**/docs/**'
+#      - 'docs/**'
+#      - '**/**.md'
+#      - '**.md'
+#      - '**/layer_tests_summary/**'
+#      - '**/conformance/**'
+#    branches:
+#      - master
+
+concurrency:
+  # github.ref is not unique in post-commit
+  group: ${{ github.event_name == 'push' && github.run_id || github.ref }}-linux-cuda
+  cancel-in-progress: true
+
+jobs:
+  Build:
+    # TODO: remove. Temporary measure to prevent the workflow from scheduling on forks.
+    if: ${{ github.repository_owner == 'openvinotoolkit' }}
+    defaults:
+      run:
+        shell: bash
+    runs-on: ubuntu-20.04-8-cores
+    container:
+      image: nvidia/cuda:11.8.0-runtime-ubuntu20.04
+    env:
+      CMAKE_BUILD_TYPE: 'Release'
+      CMAKE_GENERATOR: 'Ninja'
+      CMAKE_CXX_COMPILER_LAUNCHER: ccache
+      CMAKE_C_COMPILER_LAUNCHER: ccache
+      CMAKE_CXX_LINKER_LAUNCHER: ccache
+      CMAKE_C_LINKER_LAUNCHER: ccache
+      BUILD_TYPE: 'Release'
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      OPENVINO_CONTRIB_REPO: ${{ github.workspace }}/openvino_contrib
+      BUILD_DIR: ${{ github.workspace }}/build
+      INSTALL_DIR: ${{ github.workspace }}/install_pkg
+      DEBIAN_FRONTEND: 'noninteractive'
+    steps:
+
+      - name: Install Prerequisites
+        run: |
+          apt update
+          apt install -y git curl git git-lfs unzip wget
+
+      - name: Clone OpenVINO
+        uses: actions/checkout@v3
+        with:
+          path: 'openvino'
+          submodules: 'true'
+
+      - name: Clone OpenVINO Contrib
+        uses: actions/checkout@v3
+        with:
+          repository: 'openvinotoolkit/openvino_contrib'
+          path: 'openvino_contrib'
+          submodules: 'true'
+
+      - name: Create Directories
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/build
+          mkdir -p $GITHUB_WORKSPACE/install_pkg
+
+      - name: Setup Python 3.11
```

Comment:
looks like we don't need Python, because it's disabled in build via `ENABLE_PYTHON=OFF`